backend/routes/mattresses.route.ts

```diff
@@ -0,0 +1,9 @@
+import express, {Express, Request, Response} from "express"
+import MattressController from "../controllers/mattresses.controller";
+
+const mattressRoutes = (app: any) => {
```

Comment:
what is `app` here?
I don't think anything should be left as type `any` unless there is a valid reason